### PR TITLE
Updates to beta overview page

### PR DESCRIPTION
**Pull request by louisadetje1**
- Grammar fixes
- More explicit information about layers including how to integrate Sentera first
- Split out prescription definition from Leaf's prescription service

### CODE REVIEW

File: docs/beta_features_overview.md [hunk 1]
```diff
@@ -5,45 +5,53 @@ sidebar_label: Overview
...
+The services available in this section may be updated as we release improvements.
 
 Currently, we are working in the following features:
 
 [1]: #machines
 
 ## Machines
 
-This product fetchs machine data from the providers (like JohnDeere, ClimateFieldView, CNHI) and store this data.
+This product fetches and stores machine data such as serial number, fuel consumption and engine hours from providers.
```
Comment (guthierry) [inline docs/beta_features_overview.md:17]: Needs to confirm, but we don't have engine hours info yet
@allanpetersoon could you please check it?

File: docs/beta_features_overview.md [hunk 2]
```diff
@@ -5,45 +5,53 @@ sidebar_label: Overview
...
  
-Currently, we are obtaining data from providers:
+Currently, we are obtaining data from the following providers:
    * JohnDeere
    * CNHI
```
Comment (guthierry) [inline docs/beta_features_overview.md:22]: We can add **Stara** as a machine provider too

File: docs/beta_features_overview.md [hunk 3]
```diff
@@ -5,45 +5,53 @@ sidebar_label: Overview
...
-This product organizes the data flow for prescriptions. For instance, the variable rate application can be done through prescription maps, which can be verified by the non-homogeneity of soil properties made in laboratory analysis. It is useful when applying lime, potassium and other products that can improve soil quality, applying only the amount needed for each area.
+What are prescriptions in agriculture? Prescriptions usually used to apply inputs at variable rates across a field, applying only the amount needed for each area. For example, a prescription could be based on soil sample results or crop monitoring data, and instructs a machine where to best distribute lime, potassium or other products that can improve soil quality or crop health.
 
-The prescription file is a zip file that should contain at least **three** files with the same name and specific extensions (**shp**, **dbf** and **shx**)
+Leaf helps organize the data flow for prescriptions. 
 
 Our supported providers are:
-* Raven Slingshot
 * ClimateFieldView
+* Raven Slingshot
```
Comment (guthierry) [inline docs/beta_features_overview.md:34]: **John Deere** is a supported provider too.

File: docs/beta_features_overview.md [hunk 4]
```diff
@@ -5,45 +5,53 @@ sidebar_label: Overview
...
+
+Note: The prescription file is a zip file that should contain at least **three** files with the same name and specific file extensions (**shp**, **dbf** and **shx**).
```
Comment (guthierry) [inline docs/beta_features_overview.md:36]: It is a description for a shapefile, I think we can mention it, something like "should contain a valid shapefile (at least thee files....)

File: docs/beta_features_overview.md (hunk 2 above)
Comment (louisadetje1) [reply docs/beta_features_overview.md:24]: Thanks for confirming, done :)

File: docs/beta_features_overview.md (hunk 1 above)
Comment (allanpetersoon) [reply docs/beta_features_overview.md]: Just to notify, what we actually have on the machine endpoints are the start and end times, referring to the machine operation

**Pull request merged**